v2/internal/controllers/dbforpostgresql_flexibleserver_crud_v1api20221201_test.go

```diff
@@ -166,6 +166,7 @@ func FlexibleServer_Configuration_20221201_CRUD(tc *testcommon.KubePerTestContex
 			Value:     to.Ptr("READ"),
 		},
 	}
+	tc.AddAnnotation(&configuration.ObjectMeta, "serviceoperator.azure.com/reconcile-policy", "detach-on-delete")
 
 	tc.CreateResourceAndWait(configuration)
 	// This isn't a "real" resource so it cannot be deleted directly
```

Comment:
Clean up comment? Or did you wanna leave these? (If so ignore the comments below asking about it on other files).

Reply:
Missed that one. Did a different search, found three dregs - now fixed.